src/containers/OrderReviewPage.jsx

```diff
@@ -274,9 +280,13 @@ class OrderReviewPage extends React.Component {
                         <CheckoutSummaryList data={orderData} isLoading={false} />
                       </div>
                       <div className="checkout-submit-container" style={{ display: 'block' }}>
```

Comment:
We should show the debug messages here as well for cases where the order cannot be completed (missing info, etc). I'll add this before merging.